test/address_book_tests.cpp

```diff
@@ -1,13 +1,52 @@
 #include <catch2/catch_test_macros.hpp>
 #include <catch2/matchers/catch_matchers_string.hpp>
 
-#include "address_book.hpp"
+#include <address_book.hpp>
+#include <synchronization.hpp>
+
+
+TEST_CASE("entries can be added and removed")
+{
+using namespace std::chrono_literals;
+
+	{
+	file_synchronization_provider first("data.db");
```

Comment:
Not sure if syntax is right. I think "{ }" are needed instead of "( )"

Reply:
Sure its correct, otherwise it would not compile. "()" is the C++98 constructor call, "{}" is the same in C++11-